front-end/.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+front-end/src
```

Comment:
To add to gitignore, add the path to a file, this will make sure no changes are pushed

Reply:
Resolved with e382cc275912e2d15c76ef149f5184880204787f